components/DropdownLight/DropdownLight.jsx

```diff
@@ -0,0 +1,72 @@
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { colors, spacing, baseFontSize } from '../shared/theme';
+// import { itemsMock } from '../../stories/DropdownLight/mock';
+
+const itemPropType = PropTypes.oneOfType([
+  PropTypes.string,
+  PropTypes.shape({
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    label: PropTypes.string,
+  }),
+]);
+
+const DropList = styled.select`
+  width: 100%;
+  height: 44px;
+  font-size: 16px;
+`;
+const OptionsItems = styled.option`
+  font-size: 16px;
+`;
+
+const DropdownLight = ({
+  disabled,
+  label,
+  placeholder,
+  items,
+  selectedItem,
+  onChange,
+  theme,
+}) => (
+  <>
+    <DropList
+      disabled={disabled}
+      onChange={onChange}
+      label={label}
+      placeholder={placeholder}
+      selectedItem={selectedItem}
+      theme={theme}
+    >
+      {items.map(item => (
+        <OptionsItems key={items}>{item}</OptionsItems>
```

Comment:
Warnings are still displayed on the console about a duplicate key due to this line. Need a new explanation on this point?